climakitae/core/data_interface.py

```diff
@@ -1397,76 +1479,35 @@ def get_data_options(
     )
 
     # Raise error for bad input from user
-    for user_input in [variable, downscaling_method, resolution, timescale, scenario]:
+    for user_input in [variable, downscaling_method, resolution, timescale]:
         if (user_input is not None) and (type(user_input) != str):
             print("Function arguments require a single string value for your inputs")
             return None
 
+    def _list(x):
+        """Convert x to a list if its not a list"""
+        if type(x) == list:
+            return x
+        elif type(x) != list:
+            return [x]
+
     d = {
-        "variable": variable,
-        "timescale": timescale,
-        "downscaling_method": downscaling_method,
-        "scenario": scenario,
-        "resolution": resolution,
+        "variable": _list(variable),
+        "timescale": _list(timescale),
+        "downscaling_method": _list(downscaling_method),
+        "scenario": _list(scenario),
+        "resolution": _list(resolution),
     }
 
-    for key, val in zip(
-        d.keys(), d.values()
-    ):  # Loop through each key, value pair in the dictionary
-        # Use the catalog to find the valid values in the list
-        valid_options = np.unique(cat_df[key].values)
-        if (
-            val == None
-        ):  # If the user didn't input anything for that key, set the values to all the valid options
-            d[key] = valid_options
-            continue  # Don't finish the loop
-        # If the input value is not in the valid options, see if you can help the user out
-        elif val not in valid_options:
-
-            # This catches any common bad inputs for resolution: i.e. "3KM" or "3km" instead of "3 km"
-            if key == "resolution":
-                try:
-                    good_resolution_input = val.lower().split("km")[0] + " km"
-                    if good_resolution_input in valid_options:
-                        print("Input " + key + "='" + val + "' is not a valid option.")
-                        print(
-                            "Returning output for closest option: '"
-                            + good_resolution_input
-                            + "'"
-                        )
-                        d[key] = [good_resolution_input]
-                        continue
-                except:
-                    pass
-
-            print("Input " + key + "='" + val + "' is not a valid option.")
+    # For the non-None inputs, check if the inputs are good
```

Comment:
Does this commented out code need to be in?

Reply:
Nope; I will delete, thanks!